### ISSUE TITLE
Baseline plugin build failure

### ISSUE THREAD

**Issue by sberyozkin** [eclipse/microprofile-jwt-auth#203]
Currently the build is failing with:
```
[WARNING] Could not transfer metadata org.eclipse.microprofile.jwt:microprofile-jwt-auth-api/maven-metadata.xml from/to ossrh (https://oss.sonatype.org/service/local/staging/deploy/maven2/): Authentication failed for https://oss.sonatype.org/service/local/staging/deploy/maven2/org/eclipse/microprofile/jwt/microprofile-jwt-auth-api/maven-metadata.xml 401 Unauthorized
[INFO] The baseline version was found to be 1.2-RC1
Downloading from ossrh: https://oss.sonatype.org/service/local/staging/deploy/maven2/org/eclipse/microprofile/jwt/microprofile-jwt-auth-api/1.2-RC1/microprofile-jwt-auth-api-1.2-RC1.jar
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/microprofile/jwt/microprofile-jwt-auth-api/1.2-RC1/microprofile-jwt-auth-api-1.2-RC1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/microprofile/jwt/microprofile-jwt-auth-api/1.2-RC1/microprofile-jwt-auth-api-1.2-RC1.jar (12 kB at 42 kB/s)
[ERROR] The bundle version change (1.2.0 to 1.2.0) is too low, the new version must be at least 1.2.1
```
I'm travelling tomorrow and I don't have time now to investigate it.
Is it related to that Warning ? FYI, I've tried to update 2 package-info files to `1.2.1` but it did not help, I really don't get what is going on.
Would appreciate if someone can have a look
CC @radcortez @rdebusscher @Emily-Jiang

**Comment by radcortez** [comment]
I can have a look.

**Comment by rdebusscher** [comment]
There is already a 1.2-RC1 on Maven central. https://mvnrepository.com/artifact/org.eclipse.microprofile.jwt/microprofile-jwt-auth-api/1.2-RC1

**Comment by Emily-Jiang** [comment]
The reason is that the current branch compares to the latest released 1.2-RC1. Since there are changes, the tool asks for update. However, we pushed the RCx just for testing purpose and we compare against the latest final release. Therefore, I have added the base to 1.1. It fixes the issue. I will follow up with bnd tools to see whether we can configure towards comparing with the latest final release instead of RCx, Mx, etc.
